script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -518,87 +614,112 @@ All I want right now is to be left alone in my room. I want to take off my glass
 [sha:c398acf406d0057078d9454f29631868ff176169]{
 -- Page 218, Offset 4561.
 -- @g　呼び出しボタンを押す。中に入る。
-@gI press the call button. I go inside.
+@g
+I press the call button. I go inside.
 }
 [sha:e992a594172d20e17c2b39ad557dd30c02625520]{
 -- Page 219, Offset 4562.
 -- @g　途端、あまりの気配にぞくりときた。
-@gAt that moment, I'm overwhelmed by her remaining presence.
+@g
+I'm immediately overwhelmed by a sense of her presence.
```

Comment:
merge these two vers imo. "I'm immediately overwhelmed by her remaining presence." Boy I wish lingering wasn't on the next line. "remaining scent" would sound terrible.

Reply:
Yeah I initially wrote lingering and then welp :aokoheh:
I don't love remaining either though, how about
> I'm immediately overwhelmed by the traces of her presence.

Reply:
works for me